actors/builtin/miner/miner_actor.go

```diff
@@ -1730,15 +1737,14 @@ func handleProvingDeadline(rt Runtime) {
 			unlockedBalance = big.Sub(unlockedBalance, penaltyFromBalance)
 			penaltyTotal = big.Sum(penaltyTotal, penaltyFromVesting, penaltyFromBalance)
 			pledgeDelta = big.Sub(pledgeDelta, penaltyFromVesting)
-
 		}
 		{
 			// Record faulty power for penalisation of ongoing faults, before popping expirations.
 			// This includes any power that was just faulted from missing a PoSt.
 			penaltyTarget := PledgePenaltyForDeclaredFault(epochReward.ThisEpochRewardSmoothed, pwrTotal.QualityAdjPowerSmoothed, deadline.FaultyPower.QA)
 			penaltyFromVesting, penaltyFromBalance, err := st.PenalizeFundsInPriorityOrder(store, currEpoch, penaltyTarget, unlockedBalance)
 			builtin.RequireNoErr(rt, err, exitcode.ErrIllegalState, "failed to unlock penalty")
-			unlockedBalance = big.Sub(unlockedBalance, penaltyFromBalance) //nolint:ineffassign
+			unlockedBalance = big.Sub(unlockedBalance, penaltyFromBalance)
```

Comment:
While reviewing #1009 I determined that the `unlockedBalance` calculated above is incorrect, as it doesn't take into account balance earmarked for burning from expired precommits.

Reply:
will fix with #1026